eslint.config.js

```diff
@@ -0,0 +1,248 @@
+        'no-native-reassign': 'error',
+        'no-negated-condition': 'error',
+        'no-negated-in-lhs': 'error',
+        'no-nested-ternary': 'error',
+        'no-new': 'error',
+        'no-new-func': 'error',
+        'no-new-object': 'error',
+        'no-new-require': 'error',
+        'no-new-wrappers': 'error',
+        'no-octal-escape': 'error',
+        'no-param-reassign': 'off',
+        'no-path-concat': 'error',
+
+        'no-plusplus': ['error', {
+            'allowForLoopAfterthoughts': true,
+        }],
+
+        'no-process-env': 'off',
+        'no-process-exit': 'error',
+        'no-proto': 'error',
+        'no-prototype-builtins': 'error',
+        'no-restricted-globals': 'error',
+        'no-restricted-imports': 'error',
+        'no-restricted-modules': 'error',
+        'no-restricted-properties': 'error',
+        'no-restricted-syntax': 'error',
+        'no-return-assign': 'error',
+        'no-return-await': 'error',
+        'no-script-url': 'error',
+        'no-self-compare': 'error',
+        'no-sequences': 'error',
+        'no-shadow': 'error',
+        'no-shadow-restricted-names': 'error',
+        'no-spaced-func': 'error',
+        'no-sync': 'error',
+        'no-tabs': 'error',
+        'no-template-curly-in-string': 'error',
+        'no-ternary': 'error',
+        'no-throw-literal': 'error',
+        'no-trailing-spaces': 'error',
+        'no-undef-init': 'error',
+        'no-undefined': 'off',
+        'no-underscore-dangle': 'off',
+        'no-unmodified-loop-condition': 'error',
+        'no-unneeded-ternary': 'error',
+        'no-unused-expressions': 'error',
+        'no-use-before-define': 'error',
+        'no-useless-call': 'error',
+        'no-useless-computed-key': 'error',
+        'no-useless-concat': 'error',
+        'no-useless-constructor': 'error',
+        'no-useless-escape': 'error',
+        'no-useless-rename': 'error',
+        'no-useless-return': 'error',
+        'no-var': 'off',
+        'no-void': 'error',
+        'no-warning-comments': 'error',
+        'no-whitespace-before-property': 'error',
+        'no-with': 'error',
+        'object-curly-newline': 'off',
+        'object-curly-spacing': 'off',
+        'object-property-newline': 'off',
+        'object-shorthand': 'off',
+        'one-var': 'off',
+        'one-var-declaration-per-line': 'error',
+        'operator-assignment': 'error',
+        'operator-linebreak': 'error',
+        'padded-blocks': 'off',
+        'prefer-arrow-callback': 'off',
+        'prefer-const': 'error',
+        'prefer-destructuring': 'error',
+        'prefer-numeric-literals': 'error',
+        'prefer-promise-reject-errors': 'error',
+        'prefer-reflect': 'off',
+        'prefer-rest-params': 'off',
+        'prefer-spread': 'error',
+        'prefer-template': 'off',
+        'quote-props': 'off',
+        'quotes': ['error', 'single'],
+        'radix': 'error',
+        'require-await': 'error',
```

Comment:
I removed "require-jsdoc": "error", cause it was completely deprecated in 2018. And it was causing errors `TypeError: Key "rules": Key "require-jsdoc": Could not find "require-jsdoc" in plugin "@".`